mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -20,12 +22,15 @@ CLASS net/minecraft/class_1856 net/minecraft/recipe/Ingredient
 	METHOD method_8089 toJson ()Lcom/google/gson/JsonElement;
 	METHOD method_8091 ofItems ([Lnet/minecraft/class_1935;)Lnet/minecraft/class_1856;
 	METHOD method_8092 ofEntries (Ljava/util/stream/Stream;)Lnet/minecraft/class_1856;
-	METHOD method_8096 createStackArray ()V
+	METHOD method_8096 cacheMatchingStacks ()V
 	METHOD method_8100 getIds ()Lit/unimi/dsi/fastutil/ints/IntList;
 	METHOD method_8101 ofStacks ([Lnet/minecraft/class_1799;)Lnet/minecraft/class_1856;
 	METHOD method_8102 fromJson (Lcom/google/gson/JsonElement;)Lnet/minecraft/class_1856;
-		ARG 0 element
+		ARG 0 jsonElement
 	METHOD method_8103 isEmpty ()Z
-	METHOD method_8105 getStackArray ()[Lnet/minecraft/class_1799;
+	METHOD method_8105 getMatchingStacks ()[Lnet/minecraft/class_1799;
 	METHOD method_8106 fromTag (Lnet/minecraft/class_3494;)Lnet/minecraft/class_1856;
 	METHOD method_8107 entryFromJson (Lcom/google/gson/JsonObject;)Lnet/minecraft/class_1856$class_1859;
+		ARG 0 jsonObject
+	METHOD test matches (Ljava/lang/Object;)Z
```

Comment:
Dont rename this method as it must be `test` due to it extending Predicate

Reply:
Why does it show as `method_xxxx` for me?

Reply:
It's because of peculiarities with java generics. You have a `test(Object)` method that delegates to a renamed `test(ItemStack)`

Reply:
And the test(Object) is un-nameable?

Reply:
Alright, removed that